### DISCUSSION TITLE
Firefox by default with the elementary theme.

### DISCUSSION THREAD

**Discussion by Ditract** [elementary/triage#356] [Ideas]
![firefox](https://github.com/elementary/triage/assets/94490205/f9fcb347-6612-4427-90f7-60608f9990b2)

**Comment by danirabbit** [comment]
Hey thanks for your submission, this is essentially a duplicate of https://github.com/orgs/elementary/discussions/114 so closing as a duplicate

Specifically regarding Firefox, we've been aware of numerous occasions where essential features will break such as scrollbars, context menus, etc. So it's really not something that we could consider as a default since we can't rely on it to be well-tested and functional
